action_server/src/action_server/actions/__init__.py

```diff
@@ -9,7 +9,7 @@
 from guide import Guide
 from gripper_goal import GripperGoal
 from hand_over import HandOver
-from inspect import Inspect
```

Comment:
Absolute imports will fix this

Reply:
That does not make sense for a `__init__.py` file

Reply:
What are the guidelines for python3?

Reply:
Normal imports can also be relative but preferred to be absolute, esp. in our codebase.

When creating a package, which we do here, such relative imports are how it should be done: https://docs.python.org/2/tutorial/modules.html#packages

Python3 didn't change in this regard AFAIK.

Reply:
I am not sure what the best solution is yet, but if we start changing names, it should be consistent and done for all actions.